.envrc

```
@@ -0,0 +1,13 @@
+if ! find_up ./.venv/checkvenv.txt; then
```

```suggestion
if [ ! -d "path/to/venv" ]; then
```
Look at the man page for the `test` command to learn what `-d` does. I'd also suggest checking if the directory exists vs a specific file because we don't know everything about the contents of the venv.